packages/components/organisms/f-registration/src/components/_tests/f-registration.integration.test.js

```diff
@@ -52,19 +61,15 @@ describe('Registration API service', () => {
 
     it('responds with 409 when request is made with e-mail in use', async () => {
         // Arrange
-        axiosMock.onPost(propsData.createAccountUrl, CONSUMERS_REQUEST_DATA).reply(409, {
+        mockFactory.setupMockResponse(httpVerbs.POST, propsData.createAccountUrl, CONSUMERS_REQUEST_DATA, 409, {
             faultId: 'e2ea5f11-f771-487a-9f80-5c6f0981890b',
```

Comment:
I see this has not changed but is this sensitive data by any chance?

Reply:
@zlatinivanov-je Nope, just a GUID 👍🏼

Reply:
Is it worth making it something obviously-not-sensitive like `00000000-0000-0000-0000-000000000000`?

Reply:
Updated GUID to make it less scary